ee-feature-pack/common/pom.xml

```diff
@@ -1889,6 +1889,15 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.jboss.spec.javax.resource</groupId>
+            <artifactId>jboss-connector-api_1.7_spec</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.spec.javax.transaction</groupId>
+            <artifactId>jboss-transaction-api_1.3_spec</artifactId>
+        </dependency>
+
```

Comment:
@tadamski 

These shouldn't be in this pom; they are part of ee-feature-pack/ee-8-api/pom.xml.

We don't want EE 8 API dependencies in this pom because the WildFly Preview feature pack in ee-9/feature-pack uses this module. It does not use ee-8-api because it doesn't want those dependencies.  This kind of content sharing between feature packs is what drives the different sub-modules under ee-feature-pack and servlet-feature-pack.

Reply:
fixed